.github/workflows/pr-any.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     - uses: actions/setup-node@v1
       with:
         node-version: '12.x'
-    - name: ${{ matrix.test }}
+    - name: ${{ matrix.step }}
```

Comment:
I am sitting here swearing - CI has been acting weird and I completely missed this :(